### ISSUE TITLE
Add Activity summary tables to Activity page

### ISSUE THREAD

**Issue by ggeisler** [pod4lib/aggregator#672]
This is a story ticket intended to keep track of a set of related work tickets, plus provide an overview of how this tickets fit into the current Activity page.

- [x] #673
- [x] #674
- [x] #675
- [x] #676
- [ ] #677
- [x] #678
- [x] #679

## Context
We currently have an Activity page, visible only to POD admins. This story involves adding the section labeled "A" in the mockup below to the existing Activity page, which currently contains the legend and table in the section labeled "B" in the mockup (the "Filter uploads" box in section "B" is described in #629).

<img width="979" alt="context" src="https://user-images.githubusercontent.com/101482/164817431-89cf7594-fd23-4b83-bc9d-3aacb711ec29.png">

**Comment by ktamaral** [comment]
Branch in progress: https://github.com/pod4lib/aggregator/tree/activity-summary

**Comment by JohnMarkOckerbloom** [comment]
I've gone through the various tickets, and while I don't have a lot to say about them at this point, they look appealing and useful!  We may well learn from experience how to adjust these for the needs of admins, but this looks like a good start.

**Comment by marlo-longley** [comment]
@ggeisler Quick question about this layout - is section B supposed to show up on every tab view? Or only the uploads tab. My working assumption has been section B will show up regardless of which tab is active. But realized I should check with you. Thanks!

**Comment by ggeisler** [comment]
@marlo-longley Yes, section B should always be there. That screenshot is to provide context for the new section A, just showing how it appears before section B on the page.

**Comment by marlo-longley** [comment]
Hi @ggeisler - wanted to get your input after a conversation I had with @corylown this morning.

We've been experimenting with different caching strategies because this page is very slow to load. Specifically, loading and paging through the uploads list (section B in your mockup) is extremely slow. We realized that caching won't work well here because it will cause data to be not up to date / live / accurate. So we want to avoid caching after all.

So we were wondering if there is a way to separate section B out from section A, so that they are actually viewed on different pages, reducing load time. There's some more context in the discussion in this PR https://github.com/pod4lib/aggregator/pull/841. This is something we can maybe chat about after standup on Monday. Thanks!
